.buildkite/dagster-buildkite/dagster_buildkite/package_spec.py

```diff
@@ -254,6 +255,11 @@ def skip_reason(self) -> Optional[str]:
         if self._skip_reason:
             return self._skip_reason
 
+        if message_contains("NO_SKIP"):
+            logging.info(f"Building {self.name} because NO_SKIP set")
+            self._should_skip = False
+            return None
```

Comment:
moved this because the way it previously worked interfered with these shenanigans 

https://github.com/dagster-io/dagster/blame/5e4ae4e6cff917a36a257e3b40cf047d6a28368f/.buildkite/dagster-buildkite/dagster_buildkite/steps/test_project.py#L11-L23